configurations/simple.yaml

```diff
@@ -11,7 +11,7 @@ simulator:
 
   # Defines the ambulances to be used in the simulation
   ambulances:
-    class: CustomAmbulanceSet
+    class: BaseSelectedAmbulanceSet
```

Comment:
Will this code work? If so, we have two different ways of writing the same potentially user-confusing thing

Reply:
I'm referring to the presence of both `class` and `(class, classpath)`

Reply:
No that code doesn't work. I didn't modify the other yamls yet.

Reply:
Okay, np.